getting_started/first_steps.md

```diff
@@ -149,9 +188,76 @@ hello world
 
 Like the `cat.ts` example, the `copy()` function here also does not make
 unnecessary memory copies. It receives a packet from the kernel and sends it
-back, without further complexity.
+back, without further complexity. -->
+
+## Putting it all together in an HTTP server
+
+One of the most common usecases for Deno is building an HTTP Server.
+
+**http_server.ts**
+
+```ts
+import { serve } from "https://deno.land/std@0.119.0/http/server.ts";
+import { urlJoin } from "https://deno.land/x/url_join/mod.ts";
```

Comment:
I wanted to show specifically an example that imports a third party module, because realistically, that's a pretty basic thing people are going to be doing. If you guys have another third party module that you think would be better for the example, feel free to suggest.